src/main/java/creativei/dao/InquiryCustomDao.java

```diff
@@ -0,0 +1,14 @@
+package creativei.dao;
+
+import creativei.entity.City;
+import creativei.entity.Inquiry;
+import creativei.enums.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface InquiryCustomDao extends JpaRepository<Inquiry,Long>{
```

Comment:
No need to extend repository dao here. Extend following-
Inquirydao extends repositoryDao and the n add inquiryDao autowire into inquiryCustomdaoimpl. Follow the pr I shared as example

Reply:
issue resolved in commit 75991e6